Lib/test/test_asyncio/test_tasks.py

```diff
@@ -506,8 +506,10 @@ async def task():
         try:
             t = self.new_task(loop, task())
             self.assertFalse(t.cancelling())
+            self.assertNotIn(" cancelling ", repr(t))
             self.assertTrue(t.cancel())
             self.assertTrue(t.cancelling())
+            self.assertIn(" cancelling ", repr(t))
             self.assertFalse(t.cancel())
```

Comment:
I'd like to also test that the cancelled bit stays on even if the task catches CancelledError and sleeps some more.

Reply:
I'll try to write such a test later. May take a few days, depending on my other duties.